models/productDao.js

```diff
@@ -1,5 +1,90 @@
 const { teaDataSource } = require("./datasource.js");
 
+const getProductList = async (
+  start,
```

Comment:
- start(offset)에 대한 기본값 설정해주세요
```suggestion
  start = 0,
```